src/RadhydroShockCGS/test_radhydro_shock_cgs.cpp

```diff
@@ -71,7 +71,7 @@ template <> struct RadSystem_Traits<ShockProblem> {
 template <> struct HydroSystem_Traits<ShockProblem> {
 	static constexpr double gamma = gamma_gas;
 	static constexpr bool reconstruct_eint = true;
-  	static constexpr int nscalars = 0;       // number of passive scalars
+  static constexpr int nscalars = 0;       // number of passive scalars
```

Comment:
can you revert this change? unnecessary whitespace changes make the commit history very noisy.